about/process/security-vulnerabilities.rst

```diff
@@ -0,0 +1,42 @@
+Security vulnerabilities
+========================
+
+We take security vulnerabilities seriously. If you've found one, the best way to report it is to create a confidential issue on our `bug tracker <https://gitlab.com/ubports/ubuntu-touch>`_. Make sure you select "This issue is confidential and should only be visible to team members with at least Reporter access" when creating the issue. After that, we'll contact you through the issue to coordinate the fix and the disclosure.
```

Comment:
The way this is written, it sounds like the most important parts of handling the issue are talking to the reporter and disclosing it. I assume that those are secondary and probably optional parts. 

How about this:

> After that, we'll get to work with the reported issue. We might contact you via the issue if there is additional information needed. And we'll determine the disclosure.

Reply:
Adopted your wording. Thank you.